src/odb/src/upf/upf.tcl

```diff
@@ -195,3 +195,64 @@ proc set_isolation { args } {
 
 
 
+sta::define_cmd_args "use_interface_cell" { \
```

Comment:
The new commands need documentation.

Reply:
the first one "use_interface_cell" is already part of upf. the area one however is ours.

Reply:
added